Mathlib/Analysis/Meromorphic/Basic.lean

```diff
@@ -259,6 +259,39 @@ variable {s t : 𝕜 → 𝕜} {f g : 𝕜 → E} {U : Set 𝕜}
   (hs : MeromorphicOn s U) (ht : MeromorphicOn t U)
   (hf : MeromorphicOn f U) (hg : MeromorphicOn g U)
 
+/--
+If `f` is meromorphic on `U`, if `g` agrees with `f` on a codiscrete subset of `U` and outside of
+`U`, then `g` is also meromorphic on `U`.
+-/
+theorem congr_codiscreteWithin (hf : MeromorphicOn f U) (h₁ : f =ᶠ[codiscreteWithin U] g)
```

Comment:
```suggestion
theorem congr_codiscreteWithin_of_eqOn_compl (hf : MeromorphicOn f U) (h₁ : f =ᶠ[codiscreteWithin U] g)
```
To me, the main lemma is the next one, so I'd take a more complicated name for this one, to free the better name `congr_codiscreteWithin` for the other one.